### ISSUE TITLE
Automatically play video on web player

### ISSUE THREAD

**Issue by SubJunk** [UniversalMediaServer/UniversalMediaServer#3666]
I think it would be good to go back to doing that. At the moment you have an extra click and it opens over the content. It would be good if it can start within the page and then the user can choose to go fullscreen or not, in my opinion. Does anyone have other ideas?

**Comment by Mik-S-UMS** [comment]
Yes I think this is best, but might be better to have an option for the user to choose whether to auto play or not. You could also add an option to autoplay next episode of a series too.
